### ISSUE TITLE
Version 0.3.2 "crashes" with Textmate 1.5.10(1623)

### ISSUE THREAD

**Issue by ghost** [topfunky/PeepOpen-Issues#224]
I recently have the problem, when pressing command+t that peepopen crashes.
The window appears, but I can't type in any text. Other than a permanent rainbow wheel of death nothing happens.

console is rather clean with only:

```
4/20/11 12:04:13 PM TextMate[77630] PeepOpen: Opening URL peepopen:///users/Clint/work/projects/monkeystolemybike?editor=TextMate
```

When I focus another window, peepopen just closes.
I tried reinstalling the App, but sadly no difference.

Any advice?

Thank you very much.

**Comment by topfunky** [comment]
It's weird that it shows lowercase "users" in the path to the file.

Are you on a case-sensitive file system? This could help me in debugging.

**Comment by ghost** [comment]
No, the system is not case-sensitive. The above mentioned folder is acessible via 'cd' or any other terminal command.
